sdx_lc/models/port.py

```diff
@@ -47,10 +49,14 @@ def __init__(
         :type state: str
         :param nni: The nni of this Port.  # noqa: E501
         :type nni: str
+        :param type: Technology and bandwidth of this physical Port.  # noqa: E501
+        :type type: str
+        :param mtu: The Maximum Transmission Unit of this Port.  # noqa: E501
+        :type mtu: int
         :param services: The services of this Port.  # noqa: E501
         :type services: Service
-        :param private_attributes: The private_attributes of this Port.  # noqa: E501
-        :type private_attributes: List[str]
+        :param private: The private of this Port.  # noqa: E501
```

Comment:
The description "The private of this Port." reads weird. Can we keep "The private attributes of this Port." (like other places) as the description?